JSDemos/Demos/PivotGrid/HeaderFilter/description.md

```diff
@@ -1,9 +1,12 @@
-A header filter allows a user to filter field values by including or excluding them from the applied filter. Click a header filter icon in the field chooser or on the field panel to open a popup menu that displays all unique field values.
+A [header filter](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/headerFilter/) allows a user to filter field values by including or excluding them from the applied filter. Click a header filter icon in the field chooser or on the field panel to open a popup menu that displays all unique field values.
```

Comment:
It's not a good thing to lead the reader away from the demo right from the start. Besides, this doesn't convey the fact that the header filter is configured in the headerFilter object.

```suggestion
A header filter allows a user to filter field values by including or excluding them from the applied filter. Click a header filter icon in the field chooser or on the field panel to open a popup menu that displays all unique field values. To display header filter icons, enable the [allowFiltering](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#allowFiltering) property.
```